torchrl/envs/libs/openspiel.py

```diff
@@ -0,0 +1,358 @@
+import importlib.util
+from typing import Dict, Optional
```

Comment:
I try to upgrade to the new `foo: Smth | SmthElse` type annotation, can we remove the `Optional` and use that instead (make sure you have a `from __future__ import annotations` for older python versions)